src/app/containers/Metadata/index.test.jsx

```diff
@@ -10,6 +12,15 @@ import { RequestContextProvider } from '../../contexts/RequestContext';
 import frontPageData from '../../../../data/igbo/frontpage/index.json';
 import liveRadioPageData from '../../../../data/korean/bbc_korean_radio/liveradio.json';
 
+const MockHelmet = ({ children }) => <>{children}</>;
+MockHelmet.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]).isRequired,
+};
+jest.mock('react-helmet', () => props => <MockHelmet {...props} />);
+
 const Container = (service, bbcOrigin, platform, data, id, pageType) => {
```

Comment:
Since you're making this more readable, can you please destructure the arguments for `Container` too.

Reply:
I was going to do that but then that makes `Container` a React component with undefined prop types! I reasoned that creating a new React component in a test file and defining it's prop types is an anti-pattern because I'll need to create tests for that component too.

Reply:
Tests all the way downnnn, jk I agree with sareh here, if were not testing Container, we equally dont need to test it with an object as an arg. We can eslint ignore the prop types complaint